modules/nw-metallb-configure-secondary-interface.adoc

```diff
@@ -58,8 +80,46 @@ spec:
 $ oc apply -f enable-ip-forward.yaml
 ----
 
-. Alternatively, you can enable IP forwarding globally by running the following command:
+.Verification
+
+.  After applying the MachineConfig, verify that the changes have been applied to the target node specified by `<node_role>` in the MachineConfig YAML by the following this procedure. 
+
+.. Enter into a debug session on the target node by running the following command: 
++
+[source,terminal]
+----
+$ oc debug node/<node-name>
+----
+This step instantiates a debug pod called `<node_name>-debug`.
+
+.. Set `/host` as the root directory within the debug shell. The debug pod mounts the host’s root file system in `/host` within the pod. By changing the root directory to `/host`, you can run binaries contained in the host’s executable paths:
```

Comment:
Remember "by running the follow command..." language.

Could move sentences 2 and 3 after the command.